.script/package-automation/arm-ttk-tests.ps1

```diff
@@ -0,0 +1,57 @@
+
+try {
+    $diff = git diff --diff-filter=d --name-only HEAD^ HEAD
+    Write-Host "List of files in PR: $diff"
+
+    $hasmainTemplateChanged = $false
+    $hasCreateUiDefinitionTemplateChanged = $false
+
+    $isChangeInSolutionsFolder = [bool]($diff | Where-Object {$_ -like 'Solutions/*'})
```

Comment:
What is this check for solutions folder?

Reply:
It checks if there are changes in Solutions folder only. If yes then it goes and check if there are changes in maintemplate and/or createUiDefinition file.

Reply:
I got that. why don't we directly check inside solutions/package etc..

Reply:
Inside of Solutions we have different solution folders like Alibab Cloud, Cisco Umbrella etc and inside of those folders we have Package folder. We cannot directly get Package folder for a solution. Also we need Solution Name as well.